code/example-project/app.js

```diff
@@ -0,0 +1,55 @@
+var path = require('path');
+
+var apos = require('apostrophe')({
+  shortName: 'final-test',
```

Comment:
If we're going to ship the example project code, shortName should be `example-project`